docs/sources/community/design-documents/2020-02-Promtail-Push-API.md

```diff
@@ -2,8 +2,8 @@
 title: Promtail Push API
 description: Promtail Push API
 aliases: 
-- /docs/loki/latest/design-documents/
-- /docs/loki/latest/community/design-documents/
+- ../design-documents/2020-02-promtail-push-api/
```

Comment:
The alias in this patch does not redirect me from http://localhost:3002/docs/loki/latest/design-documents/2020-02-promtail-push-api/ to http://localhost:3002/docs/loki/latest/design-documents/2020-02-promtail-push-api/.

Instead I believe it should be:
```suggestion
- ../../design-documents/2020-02-promtail-push-api/
```